### PR TITLE
Add acceptance test document

### PR DESCRIPTION
**Pull request by tzumainn**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202062295771285

### CODE REVIEW

File: acceptance-tests.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    * _As a new user, I should be able to create an account for myself in the NERC._
+    * <span style="text-decoration:underline;">Steps</span>
+        1. A prospective user follows the steps documented in [https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/](https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/) to create an account on the NERC.
+2. **Remove a user**
+    * _As an administrator, I should be able to remove a user, labels association with the user, and user identity from the cluster._
+       * _(This step is generally unnecessary, given that a user that doesn’t have authorization to any project, cannot access any allocations, therefore removing a user from allocations is generally enough.)_
+    * <span style="text-decoration:underline;">Steps</span>
+        1. A NERC admin uses a private runbook to disable/ deactivate a user from KeyCloak.
+        2. _(Optional)_ After that is done, the NERC admin uses the OpenShift CLI and runs the following:
+            1. oc delete user &lt;username>
```
Comment (joachimweyl) [inline acceptance-tests.md]: should this and the line below be in code blocks? `oc delete user <username>`

File: acceptance-tests.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-started/user-onboarding-on-NERC/](https://nerc-project.github.io/nerc-docs/get-started/user-onboarding-on-NERC/)
+
+
+## Managing Users
+
+
+
+1. **Request a new account**
+    * _As a new user, I should be able to create an account for myself in the NERC._
```
Comment (joachimweyl) [inline acceptance-tests.md:11]: What was the reasoning to use `*` instead of `1.` here?

File: acceptance-tests.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+
+1. **Online Documentation**
+    * _As an administrator or a user, I should be able to perform any routine operations by referring to online documentation which lists the steps that need to be taken to complete an operation._
+    * <span style="text-decoration:underline;">Steps</span>
+        1. Administrator/user accesses documentation at:
+            1. [https://nerc-project.github.io/nerc-docs/](https://nerc-project.github.io/nerc-docs/)
+
+
+## Sure Hardware Management
```
Comment (joachimweyl) [inline acceptance-tests.md]: What does `Sure` represent here?

**Comment by joachimweyl** [comment]
I added a line to the top of the [google doc](https://docs.google.com/document/d/18AUzxuFWKeNOO6UFgze6Tm1mF1Gz-UIMj8Wqu6lyxcc/edit?usp=sharing). please update the link to point to the Markdown file once it is live. For now, it points to this PR.

File: acceptance-tests.md (hunk 2 above)
Comment (tzumainn) [reply acceptance-tests.md:11]: Ah, my thought was that it's not a step in the acceptance tests, so I didn't want to imply that it contained an action to be executed.

File: acceptance-tests.md (hunk 1 above)
Comment (tzumainn) [reply acceptance-tests.md]: Yep - fixed!

File: acceptance-tests.md (hunk 3 above)
Comment (tzumainn) [reply acceptance-tests.md]: Hm, probably a typo - removed!

**Review by joachimweyl — APPROVED** [review]
Matches the Google doc except where I made suggestions of small changes to make it more accurate.

**Pull request merged**